pkg/kn/plugin/manager.go

```diff
@@ -125,6 +105,10 @@ func (manager *Manager) AppendPlugin(plugin Plugin) {
 	InternalPlugins = append(InternalPlugins, plugin)
 }
 
+func (manager *Manager) GetInternalPlugins() PluginList {
```

Comment:
Usually you don't need a getter if you just return a exported variable. So, I would make either `InternalPlugins` private (and then call the method just `InternalPlugins()` which is probably more idiomatic go), or just access the public exported field when using the InternalPlugins (strange feeling for us Java guys, I know ;-)

Reply:
Yep, it still kind of nicer to have a getter. I'll get rid of it though. From the compatibility perspective we can't make `InternalPlugins` private due to inlined plugins appends.